app/models/school.rb

```diff
@@ -80,7 +80,14 @@ class School < ApplicationRecord
                   against: %i[name postcode],
                   using: { trigram: { word_similarity: true } }
 
+  PRIMARY_PHASE = "Primary".freeze
+  SECONDARY_PHASE = "Secondary".freeze
+
   def organisation_type
     "school"
   end
+
+  def primary_or_secondary_only?
```

Comment:
Does this need the `_only` suffix?

Reply:
As "All-through" and possibly some other phases class as "Primary and Secondary", I figured the `only` suffix makes that explicit